Create Task 0

GinikaJ opened the pull request.

GinikaJ commented:
@jafsia Thank you somuch for your help. I have deleted the file with the conflict and added a new one. Looking forward to your feedback.

jafsia commented:
Good

Le mer. 20 avr. 2022 à 11:57, Ginika Jemeni ***@***.***> a
écrit :

> @jafsia <https://github.com/jafsia> Thank you somuch for your help. I
> have deleted the file with the conflict and added a new one. Looking
> forward to your feedback.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Mboalab/Mboalab_Outreachy-May-Aug-2022/pull/92#issuecomment-1103799109>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQWYDSWAKPFKIIKNSCR4DGTVF7PKDANCNFSM5T3UIBVQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

GinikaJ commented:
> Good Le mer. 20 avr. 2022 à 11:57, Ginika Jemeni ***@***.***> a écrit :
> […](#)
> @jafsia <https://github.com/jafsia> Thank you somuch for your help. I have deleted the file with the conflict and added a new one. Looking forward to your feedback. — Reply to this email directly, view it on GitHub <[#92 (comment)](https://github.com/Mboalab/Mboalab_Outreachy-May-Aug-2022/pull/92#issuecomment-1103799109)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQWYDSWAKPFKIIKNSCR4DGTVF7PKDANCNFSM5T3UIBVQ> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Thank you very much. So it's a go to be merged?

GinikaJ commented:
Thank you so much sir.

On Wed, 20 Apr 2022 at 12:21, JAFSIA Elisee ***@***.***>
wrote:

> Good
>
> Le mer. 20 avr. 2022 à 11:57, Ginika Jemeni ***@***.***> a
> écrit :
>
> > @jafsia <https://github.com/jafsia> Thank you somuch for your help. I
> > have deleted the file with the conflict and added a new one. Looking
> > forward to your feedback.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/Mboalab/Mboalab_Outreachy-May-Aug-2022/pull/92#issuecomment-1103799109
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AQWYDSWAKPFKIIKNSCR4DGTVF7PKDANCNFSM5T3UIBVQ
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Mboalab/Mboalab_Outreachy-May-Aug-2022/pull/92#issuecomment-1103818198>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYO7ZAGXU445QRZ2B2V6E43VF7SBZANCNFSM5T3UIBVQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
